README.md

```diff
@@ -1,7 +1,35 @@
-# Settings View package [![Build Status](https://travis-ci.org/atom/settings-view.svg?branch=master)](https://travis-ci.org/atom/settings-view)
+# Settings View [![Build Status](https://travis-ci.org/atom/settings-view.svg?branch=master)](https://travis-ci.org/atom/settings-view)
 
-Edit config settings, install packages, and change themes in Atom.
+Edit config settings, install packages, and change themes all from within Atom.
 
-Open by pressing `cmd-,` or selecting the _Atom > Preferences..._ menu.
+## Usage
+You can open the Settings View by navigating to _Atom > Preferences_ (Mac), _File > Preferences_ (Windows), or _Edit > Preferences_ (Linux).
 
-![screen shot 2015-01-05 at 8 08 18 pm](https://cloud.githubusercontent.com/assets/1476/5624241/d29ffbe6-9516-11e4-8bfe-e91623977bc7.png)
+In order to install new packages and themes, simply click on the _Install_ section located on the left-hand side.
```

Comment:
Maybe :scissors: "simply" and "located"? I think the sentence doesn't lose any meaning without those.